src/Symfony/Component/Process/ExecutableFinder.php

```diff
@@ -75,7 +75,7 @@ public function find($name, $default = null, array $extraDirs = array())
         $suffixes = array('');
         if ('\\' === DIRECTORY_SEPARATOR) {
             $pathExt = getenv('PATHEXT');
-            $suffixes = $pathExt ? explode(PATH_SEPARATOR, $pathExt) : $this->suffixes;
+            $suffixes = array_merge($suffixes, $pathExt ? explode(PATH_SEPARATOR, $pathExt) : $this->suffixes);
```

Comment:
Here we are merging the Windows suffixes with the previously-defined empty suffix on line 75, instead of overwriting the empty suffix list as previously.